client/invoke.go

```diff
@@ -9,6 +9,14 @@ import (
 	"github.com/pkg/errors"
 )
 
+// ServiceContent the service invocation content
+type ServiceContent struct {
```

Comment:
nit: The naming doesn't look self-descriptive :) 

Can we find the better naming ? DataContent ? ServiceInvocationContent ?

Reply:
Agree, `DataContent` reads better and more descriptive